src/ffi/pystate.rs

```diff
@@ -21,43 +25,45 @@ extern "C" {
     #[cfg_attr(docsrs, doc(cfg(not(PyPy))))]
     pub fn PyInterpreterState_Delete(arg1: *mut PyInterpreterState);
 
-    #[cfg(all(Py_3_9, not(PyPy)))]
+    #[cfg(all(any(not(Py_LIMITED_API), Py_3_9), not(PyPy)))]
     #[cfg_attr(docsrs, doc(all(Py_3_9, not(PyPy))))]
-    pub fn PyInterpreterState_Get() -> *mut PyInterpreterState;
+    pub fn PyInterpreterState_Get(arg1: *mut PyInterpreterState) -> *mut PyInterpreterState;
 
-    #[cfg(all(Py_3_8, not(PyPy)))]
+    #[cfg(all(any(not(Py_LIMITED_API), Py_3_8), not(PyPy)))]
     #[cfg_attr(docsrs, doc(all(Py_3_8, not(PyPy))))]
-    pub fn PyInterpreterState_GetDict() -> *mut PyObject;
+    pub fn PyInterpreterState_GetDict(arg1: *mut PyInterpreterState) -> *mut PyObject;
 
-    #[cfg(all(Py_3_7, not(PyPy)))]
+    #[cfg(all(any(not(Py_LIMITED_API), Py_3_7), not(PyPy)))]
     #[cfg_attr(docsrs, doc(all(Py_3_7, not(PyPy))))]
-    pub fn PyInterpreterState_GetID() -> i64;
+    pub fn PyInterpreterState_GetID(arg1: *mut PyInterpreterState) -> i64;
```

Comment:
Similar point about these having different names before they were added to the limited API.

Reply:
Changed as suggested.